integreat_compass/cms/fixtures/test_data.json

```diff
@@ -1 +1,1087 @@
-[]
+[
+  {
+    "model": "cms.language",
+    "pk": 1,
+    "fields": {
+      "native_name": "Englisch",
+      "english_name": "English"
+    }
+  },
+  {
+    "model": "cms.language",
+    "pk": 2,
+    "fields": {
+      "native_name": "Deutsch",
+      "english_name": "German"
+    }
+  },
+  {
+    "model": "cms.offerversion",
+    "pk": 1,
+    "fields": {
+      "offer": 1,
+      "offer_version_date": "2023-05-20T15:21:17.030Z",
+      "title": "Deutsch für Anfänger - Intensivkurs",
+      "description": "Dieser Intensivkurs bietet einen umfassenden Einstieg in die deutsche Sprache für Anfänger. Sie werden grundlegende Konversation, Grammatik und Vokabeln lernen, um sich im Alltag auf Deutsch verständigen zu können.",
+      "cost": 150,
+      "language": 2
+    }
+  },
+  {
+    "model": "cms.offerversion",
+    "pk": 2,
+    "fields": {
+      "offer": 2,
+      "offer_version_date": "2023-05-20T15:26:40.367Z",
+      "title": "Fortgeschrittenes Deutsch - Konversationskurs für die Arbeitswelt",
+      "description": "In unserem Konversationskurs \"Fortgeschrittenes Deutsch\" legen wir den Fokus auf die mündliche Kommunikation und das sprachliche Selbstbewusstsein in der Arbeitswelt. Der Kurs richtet sich an Personen, die bereits über grundlegende Deutschkenntnisse verfügen und ihre Sprachfertigkeiten gezielt für den beruflichen Kontext weiterentwickeln möchten.\r\n\r\nIm Kurs werden verschiedene Themen behandelt, die typische Situationen im Arbeitsumfeld widerspiegeln, wie zum Beispiel Bewerbungsgespräche, Präsentationen, Geschäftskorrespondenz und Teamarbeit. Durch gezielte Übungen und praxisnahe Rollenspiele werden Sie lernen, Ihre sprachlichen Fähigkeiten in diesen Situationen zu verbessern und anzuwenden.\r\n\r\nUnsere erfahrenen Lehrkräfte legen Wert auf eine interaktive Lernumgebung, in der Sie sowohl in Gruppenaktivitäten als auch in Einzelgesprächen Ihre Kommunikationsfähigkeiten stärken können. Darüber hinaus erhalten Sie wertvolles Feedback und individuelle Unterstützung, um Ihre sprachlichen Schwächen zu überwinden und Ihr Sprachniveau auf eine höhere Stufe zu heben.",
+      "cost": 250,
+      "language": 2
+    }
+  },
+  {
+    "model": "cms.user",
+    "pk": 1,
+    "fields": {
+      "password": "pbkdf2_sha256$600000$PA2SmRUyUUxufQl205scvB$BlFZuqJHPxUQwRnEGsuExRR5dYUoPXD7V559Jcfczrw=",
```

Comment:
What would this be decoded?

Reply:
Ah whoops, sorry. It's `root1234`

Reply:
Maybe we should document this somewhere?

Reply:
I guess you already documented this in https://github.com/digitalfabrik/integreat-compass/pull/18/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R33?
But yes, we could also add it to the [documentation](https://integreat-compass.readthedocs.io/en/latest/dev-server.html)...